src/wms/src/test/java/org/geoserver/wms/wms_1_1_1/GetMapIntegrationTest.java

```diff
@@ -2390,6 +2390,24 @@ public void testMultiLanguageStyle() throws Exception {
         ImageAssert.assertEquals(new File(urlPng.toURI()), image, 500);
     }
 
+    @Test
+    public void testEmptyLanguageStyle() throws Exception {
+        Catalog catalog = getCatalog();
+        LayerInfo places = catalog.getLayerByName(getLayerId(MockData.NAMED_PLACES));
+        StyleInfo multiLangStyle = catalog.getStyleByName("multiLanguageStyle");
+        places.getStyles().add(multiLangStyle);
+        catalog.save(places);
+        String url =
+                "wms?LAYERS="
+                        + places.getName()
+                        + "&STYLES=multiLanguageStyle&FORMAT=image%2Fpng"
+                        + "&SERVICE=WMS&VERSION=1.1.1&REQUEST=GetMap&SRS=EPSG%3A4326&WIDTH=256"
+                        + "&HEIGHT=256&BBOX=0.0000,-0.0020,0.0035,0.0010";
+        BufferedImage image = getAsImage(url, "image/png");
+        URL urlPng = getClass().getResource("nolang_result.png");
+        ImageAssert.assertEquals(new File(urlPng.toURI()), image, 500);
```

Comment:
A threshold of 500 is a big one, for an image that does not have any label in it. The line rendering should only change slightly across platforms (thinner or thicker, antialiasing done in a different way). I'd go for a smaller  value, like 50?

Reply:
fixed.